src/main/java/gf2t/GF2_192.java

```diff
@@ -509,7 +509,7 @@ public static void power2To2ToK(GF2_192 res, GF2_192 z, int k) {
             // so that's what we compute.
             // Note that we have no precomputed table for k=7 (i.e., 2^128), because we don't expect
             // this to be needed -- only up to k=6 is used in inversion.
-            // Here's the proof: let m = 64. 2^{2^k} mod (2^{192}-1) = 2^{mn} mod (2^{3m}-1) for n = 2^{k-6}.
+            // Here's the proof: val m = 64. 2^{2^k} mod (2^{192}-1) = 2^{mn} mod (2^{3m}-1) for n = 2^{k-6}.
```

Comment:
@greenhat please rollback this change

Reply:
@ergomorphic Thanks for spotting it. Bulk search & replace gone wrong. :) Will fix in my #248 PR.